frontend/public/components/app.jsx

```diff
@@ -215,9 +216,12 @@ class App_ extends React.PureComponent {
           <DetectNamespace>
             {contextProviderExtensions.reduce(
               (children, e) => (
-                <EnhancedProvider key={e.uid} {...e.properties}>
-                  {children}
-                </EnhancedProvider>
+                <>
+                  <EnhancedProvider key={e.uid} {...e.properties}>
+                    {children}
+                  </EnhancedProvider>
+                  <FeatureFlagExtensionLoader />
```

Comment:
```suggestion
                  <FeatureFlagExtensionLoader />
                  <EnhancedProvider key={e.uid} {...e.properties}>
                    {children}
                  </EnhancedProvider>
```

Reply:
@christianvogt @nemesis09 The order he is not the reason. It is just added with every loop one time to the render tree.

I see two options to render it just once:

1. Add it in _App render() to the `content` variable.
    Con: It's not really a content
    Pro: The hook(s) then has access to all contexProviderExtensions!

```tsx
        </QuickStartDrawer>
        <ConsoleNotifier location="BannerBottom" />
        <FeatureFlagExtensionLoader />
      </>
    );
```

**OR**

2. Add it as a sibling to the `contextProviderExtensions.reduce`.
    * Con: The hook(s) can not access data which are provided by the `contextProviderExtensions`

```tsx
        <DetectPerspective>
          <DetectNamespace>
            <FeatureFlagExtensionLoader />
            {contextProviderExtensions.reduce(
              (children, e) => (
                <EnhancedProvider key={e.uid} {...e.properties}>
                  {children}
                </EnhancedProvider>
              ),
              content,
            )}
          </DetectNamespace>
        </DetectPerspective>
        <DetectLanguage />
```